userguide/storage.rst

```diff
@@ -1843,6 +1843,14 @@ To offline, online, or or replace the device, see
    | S.M.A.R.T. extra options     | string    | ✓          | Enter additional `smartctl(8) <https://www.smartmontools.org/browser/trunk/smartmontools/smartctl.8.in>`__  options.     |
    |                              |           |            |                                                                                                                          |
    +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Critical                     | string    |            | Enter a threshold temperature in Celsius. If the drive temperature is higher than this value, a :literal:`LOG_CRIT`      |
+   |                              |           |            | level log entry is created and an email is sent. Entering :literal:`0` disables this check.                              |
+   +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Difference                   | string    |            | Report if the temperature of a drive has changed by this many degrees Celsius since the last report. Entering            |
```

Comment:
```suggestion
   | Difference                   | string    |            | Report if the temperature of a drive has changed by this many degrees Celsius since the last report.                     |
```